src/components/appliedRules/PersonaScore.jsx

```diff
@@ -52,10 +56,101 @@ const PersonaScore = (props) => {
     };
 
     treatResults(newPersona);
-  }, [resultsScore]);
+  }, [resultsScore, isSwitchToggle]);
 
   useEffect(() => {
-    console.log(isSwitchToggle);
+    const newPersona = {
+      color: [],
+      brand: [],
+      gender: [],
+      hierarchical0: [],
+      hierarchical1: [],
+      hierarchical2: [],
+      category: [],
+      categories: [],
+    };
+    if (!isSwitchToggle) {
```

Comment:
Add the fact than if we used the personaFilters at the place of the event sending we can switch the button, by default it's true
@matthew1809